src/index.ts

```diff
@@ -108,15 +108,6 @@ module.exports = class Plugin {
         // Specify known package aliases
         alias['lwc'] = require.resolve('@lwc/engine-dom')
         alias['wire-service'] = require.resolve('@lwc/wire-service')
-        // the 'main' property for @lwc/synthetic-shadow refers to a file that
-        // simply logs an error message. This needs to be fixed up to directly
-        // specify the actual implementation, which lives under
-        // /dist/synthetic-shadow.js. Note: this depends on the internal file
-        // structure of this component, and there is a little fragile, if the
-        // module ever changes where this implementation lives.
-        alias['@lwc/synthetic-shadow'] = require.resolve(
-            '@lwc/synthetic-shadow/dist/synthetic-shadow.js'
-        )
```

Comment:
Removing this is not really the right solution, because it will break for older versions of LWC. And I don't think `lwc-webpack-plugin` can safely mandate a particular LWC version (or if it did, it'd be a breaking change).

A more robust solution may be a `try/catch` to resolve `@lwc/synthetic-shadow/dist/synthetic-shadow.js` and then falling back to resolving `@lwc/synthetic-shadow`.

Reply:
100% good to NOT break older versions.

added in a `try`/`catch` to keep supporting older versions. left the catch to not alias as the `@lwc/synthetic-shadow` package should be able to resolve itself as of v3.